src/ecalc/cli/tests/snapshots/test_app/test_full_simplified_json/results.json

```diff
@@ -262,7 +262,6 @@
     {
       "componentType": "COMPRESSOR",
       "component_level": "CONSUMER",
-      "parent": "e852d5d4b9101f154db921ece82a697c",
```

Comment:
this is not supposed to be removed afaik

Reply:
Fixed when adding parent to compressor subcomponent